### ISSUE TITLE
calibration of ESM-RCM chains

### ISSUE THREAD

**Issue by mathause** [MESMER-group/mesmer#113]
@woodhome23 is working on calibrating and emulating regional climate models. This adds the complexity that there are combinations of ESMs & RCMs:

```
CESM - RCA4
AWI - RCA4
CESM - CLM
```

Now it could be desirable to calibrate on each combination individually, or groups of GCM or groups of RCM. Further, the RCMs need the GCM's global mean temperature.

I am not sure if this adds additional difficulties for the design or if this would fall into place naturally...

**Comment by woodhome23** [comment]
@mathause yes as far as I understand they do have that. It is basically four scenarios that are available 1.) historical 2.) rcp2.6 3.) rcp4.5 and 4.) rcp8.5
